env/Lib/site-packages/PIL/ImageMath.py

```
@@ -0,0 +1,83 @@
+# 1999-02-15 fl   Original PIL Plus release
+# 2005-05-05 fl   Simplified and cleaned up for PIL 1.1.6
+# 2005-09-12 fl   Fixed int() and float() for Python 2.4.1
+#
+# Copyright (c) 1999-2005 by Secret Labs AB
+# Copyright (c) 2005 by Fredrik Lundh
+#
+# See the README file for information on usage and redistribution.
+#
+from __future__ import annotations
+
+import builtins
```

Ruff (0.8.2)

`builtins` imported but unused

Remove unused import: `builtins`

(F401)